backend/qingstor/upload.go

```diff
@@ -348,7 +348,7 @@ func (mu *multiUploader) abort() error {
 		req := qs.AbortMultipartUploadInput{
 			UploadID: uploadID,
 		}
-		fs.Debugf(mu, "Tried to abort a multi-part")
+		fs.Debugf(mu, "Aborting multi-part object %s", uploadID)
```

Comment:
This should say (note extra `*`)

    fs.Debugf(mu, "Aborting multi-part object %s", *uploadID)
￼
Then the build will complete!  I'd use `%q` instead of `%s` also.

Reply:
I've push forced with *, but appveyyor wasn't happy.
Trying again with %q !